src/index.html

```diff
@@ -7,6 +7,18 @@
   </head>
   <body>
     <canvas hidden id="input-image" width="28px" height="28px"></canvas>
+    <select id="image-selection">
+      <option selected value="https://raw.githubusercontent.com/UW-CSE442-WI20/A3-convolutional-neural-networks/michan4-v1/Images/0.png">0</option>
+      <option value="https://raw.githubusercontent.com/UW-CSE442-WI20/A3-convolutional-neural-networks/michan4-v1/Images/1.png">1</option>
+      <option value="https://raw.githubusercontent.com/UW-CSE442-WI20/A3-convolutional-neural-networks/michan4-v1/Images/2.png">2</option>
+      <option value="https://raw.githubusercontent.com/UW-CSE442-WI20/A3-convolutional-neural-networks/michan4-v1/Images/3.png">3</option>
+      <option value="https://raw.githubusercontent.com/UW-CSE442-WI20/A3-convolutional-neural-networks/michan4-v1/Images/4.png">4</option>
+      <option value="https://raw.githubusercontent.com/UW-CSE442-WI20/A3-convolutional-neural-networks/michan4-v1/Images/5.png">5</option>
+      <option value="https://raw.githubusercontent.com/UW-CSE442-WI20/A3-convolutional-neural-networks/michan4-v1/Images/6.png">6</option>
+      <option value="https://raw.githubusercontent.com/UW-CSE442-WI20/A3-convolutional-neural-networks/michan4-v1/Images/7.png">7</option>
+      <option value="https://raw.githubusercontent.com/UW-CSE442-WI20/A3-convolutional-neural-networks/michan4-v1/Images/8.png">8</option>
+      <option value="https://raw.githubusercontent.com/UW-CSE442-WI20/A3-convolutional-neural-networks/michan4-v1/Images/9.png">9</option>
+    </select>
```

Comment:
Should we change those URL's to point to master?

Reply:
Just changed them